coder/coder
# fix(site): Upload template files on template version editor

How to QA: 
- Go to any template, preferably a template you can break.
- At the bottom, select a version
- Click on the edit button or add `/edit` at the end of the URL
- Edit any file, preferably with a comment, and build it
- If that works as expected - with a success message - the goal was achieved.
- You can also navigate back to the template page and access the latest version created by you, click on edit, and check if all the changes you made are still there.

**Strategy**
- The strategy was to append all the non-supported files in the tar first and after, add the ones from the editor file tree. Why? By doing that, we can rely on the file tree to append or not the editable files.
- I'm also planning to remove untar dependency to use the util we have, but probably it is also going to be done in a next PR

**Testing**
Instead of testing the entire flow, I can try to extract the "update template version files" to test only this in isolation. I don't know how hard is but it sounds valuable. I'm probably going to do that in the next PR.
